velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -1142,7 +1176,7 @@ void JoinFuzzer::verify(core::JoinType joinType) {
 
   shuffleJoinKeys(probeKeys, buildKeys);
 
-  const auto defaultPlan = makeDefaultPlan(
+  auto defaultPlan = makeDefaultPlan(
```

Comment:
Can you rebase and skip if it is merge join? I think Presto doesn't support merge join? Can you help check? Thanks!